### DISCUSSION TITLE
打开clash后只有一台pc的流量被代理，ios设备都无法访问外网

### DISCUSSION THREAD

**Discussion by nnoyuwan** [vernesong/OpenClash#3228] [Q&A]
访问控制没有做任何配置，请教可能是什么问题

**Comment by vernesong** [comment]
控制台能看到流量吗，确认DNS劫持是否成功

**Reply by nnoyuwan** [reply]
![image](https://user-images.githubusercontent.com/31734364/234301907-2276ed79-19db-4c21-82c0-123cd9ab8957.png)
![image](https://user-images.githubusercontent.com/31734364/234303148-5564a9c6-891b-4989-be5d-6cc9a55303be.png)
只有pc 192.168.31.13这个流量会走openclash，然后dhcp只有这个设备显示被分配了ip

**Comment by vernesong** [comment]
wifi检查是否是ap的问题
